server/src/main/java/io/druid/curator/CuratorConfig.java

```diff
@@ -66,4 +69,14 @@ public void setEnableCompression(Boolean enableCompression)
   {
     this.enableCompression = enableCompression;
   }
+
+  public boolean getEnableAcl()
+  {
+    return enableAcl;
+  }
+
+  public void setEnableAcl(Boolean enableAcl)
+  {
+    this.enableAcl = enableAcl;
```

Comment:
@fjy mentioned that having `Boolean` will reject the configuration in the event of fat-fingering or omitting the value